### PR TITLE
Example for migrating to use sessionSigs instead of authSigs

### PR DESCRIPTION
**Pull request by Aaryan-R-S**
### Fixes Issue: [LIT-2532](https://linear.app/litprotocol/issue/LIT-2532/examples-for-migrating-to-use-sessionsigs-from-authsigs)

# Description

Lot's of folks simply use `authSigs` which don't have capabilities. But since we now require `capacityDelegationAuthSig` they need to use `sessionSigs`

Create a before & after example explaining all the new changes

Trying to update the docs for it!

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Introducing new feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

Link to the relevant updated sections in the Netlify preview

- [ ] Link 1: [description of change]
- [ ] Link 2: [...]

# Checklist:

General
- [ ] I have performed a self-review of my code
- [ ] I have fixed all grammar issues (can use an AI tool to check), and explanations are in active voice
- [ ] I have checked the additions are concise
- [ ] Language is consistent with existing documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published (ie. SDK changes, node dependencies)

If I have added a new concept, I have
- [ ] included a beginner friendly explanation
- [ ] included a basic technical introduction and code sample
- [ ] new terms are defined, both in relevant new pages and in the glossary

### CODE REVIEW

**Review by glitch003 — APPROVED** [review]
i modified this a bit but i think it's good to go now.  thanks @Aaryan-R-S

File: docs/concepts/capacity-credits-concept.md [hunk 1]
```diff
@@ -34,9 +34,82 @@ Alice can create a session capability object that specifies the ability to Authe
...
+
+```javascript
+  const litNodeClient = new LitNodeClient({
+      litNetwork: "manzano",
+      checkNodeAttestation: true,
+  });
+  
+  await litNodeClient.connect();
+  const authNeededCallback = async ({ resources, expiration, uri }) => {
+    // you can change this resource to anything you would like to specify
```
Comment (Adarsh-Kumar28) [inline docs/concepts/capacity-credits-concept.md:48]: Provide link to the docs page that explains the Capabilities model

**Review by Adarsh-Kumar28 — CHANGES_REQUESTED** [review]
Since users who haven't use sessionSigs don't know about a lot of concepts related to it. How about adding some links to the relevant docs pages for SIWE, capabilities, etc?

**Review by Adarsh-Kumar28 — APPROVED** [review]
LGTM!

**Pull request merged**